### PR TITLE
Change wallet-api imports so it doesn't load wasm

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Fix so inject script doesn't cause errors when you use websites that dont allow wasm scripts.

## Changes

wallet-api now imports directly from common-sdk to avoid initializing the wasm module.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
         "react-virtualized-auto-sizer": "^1.0.6",
         "react-window": "^1.8.7",
         "react-window-infinite-loader": "^1.0.8",
+        "stream": "^0.0.2",
```
Comment (orhoj) [inline packages/browser-wallet/package.json:37]: Why is this needed now?

Comment (orhoj) [reply packages/browser-wallet/package.json:37]: I guess we don't get it indirectly from common-sdk or something.

Comment (shjortConcordium) [reply packages/browser-wallet/package.json:37]: Yeah we get it from common-sdk through the api now.

**Comment by orhoj** [comment]
When I connect to a dApp (piggybank), then I get this error:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'call')
```

**Comment by shjortConcordium** [comment]
> When I connect to a dApp (piggybank), then I get this error:
> 
> ```
> Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'call')
> ```

Turns out the "stream" package is **_very_** outdated.

**Review by orhoj — APPROVED** [review]
